conf/minifi.properties

```diff
@@ -53,7 +53,7 @@ nifi.database.content.repository.directory.default=${MINIFI_HOME}/content_reposi
 #nifi.c2.rest.url.ack=
 nifi.c2.root.classes=DeviceInfoNode,AgentInformation,FlowInformation
 ## heartbeat 4 times a second
-nifi.c2.agent.heartbeat.period=250
```

Comment:
defaults to 3 seconds.

Reply:
Please edit the comment above to reflect this!

Reply:
Ah, gotcha, thanks!